build_system.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 set -e
 
-UBUNTU_BASE_URL="http://cdimage.ubuntu.com/ubuntu-base/releases/20.04/release"
-UBUNTU_FILE="ubuntu-base-20.04.1-base-arm64.tar.gz"
+UBUNTU_BASE_URL="http://cdimage.ubuntu.com/ubuntu-base/releases/24.04/release"
+UBUNTU_FILE="ubuntu-base-24.04-base-arm64.tar.gz"
```

Comment:
Probably discussed somewhere already, but why dont we just use the Ubuntu docker as the base "FROM ubuntu:24.04"